bootstrap/kubeadm/internal/ignition/clc/clc.go

```diff
@@ -96,6 +96,8 @@ systemd:
         Description=kubeadm
         # Run only once. After successful run, this file is moved to /tmp/.
         ConditionPathExists=/etc/kubeadm.yml
+        After=network.target
+        After=containerd.service
```

Comment:
This assumes that containerd will always be used as a CRI, which I don't think is true. I'm pretty sure cri-o can also be used, so this would be a breaking change for such users.

I think this modification should be done via cluster template, as it allows adding additional unit overrides easily.